### PR TITLE
Deliver regenerated release.yaml created from the updated build.gradles

### PR DESCRIPTION
**Pull request by jadecarino**
This change was meant to be added as part of [this PR](https://github.com/galasa-dev/managers/pull/990) but I forgot to add this file.

### CODE REVIEW

File: release.yaml [hunk 1]
```diff
@@ -152,19 +152,19 @@ managers:
 
   - artifact: dev.galasa.liberty.manager
     version: 0.21.0
-    obr:          true
-    mvp:          true
-    bom:          true
-    javadoc:      true
-    isolated:     true
-    codecoverage: true
+    obr:          false
```
Comment (techcobweb) [inline release.yaml:155]: This doesn't look right ?

File: release.yaml [hunk 2]
```diff
@@ -476,19 +476,19 @@ managers:
 
   - artifact: dev.galasa.windows.manager
     version: 0.21.0
```
Comment (techcobweb) [inline release.yaml:478]: Not sure what the windows manager does ? You are pulling it out of everywhere intentionally ?

**Review by techcobweb — APPROVED** [review]

File: release.yaml (hunk 1 above)
Comment (jadecarino) [reply release.yaml:155]: The liberty manager is completely unimplemented it looks like it was started and forgotten about - it just contains several unimplemented interfaces. It shouldn't really have been delivered or released in its current state. Thought it best to remove from everywhere. Luckily it's not mentioned on the website.

File: release.yaml (hunk 2 above)
Comment (jadecarino) [reply release.yaml:478]: Thank you for noticing I had in my notes in the README.md changes I made earlier this year that the windows manager looked minimally implemented so thought it best to remove but I've just found when looking again that its a dependency of another two managers, so I've added it back into the OBR, BOM and Isolated as those two managers are in isolated and all dependencies need to be.

File: release.yaml (hunk 1 above)
Comment (jadecarino) [reply release.yaml:155]: It was however a dependency of the zos liberty manager but none of its classes were used, so I have removed it as a dependency of that manager so there are no build issues due to the liberty manager being removed from OBR.

**Pull request merged**